src/main/java/com/spotify/futures/FuturesExtra.java

```diff
@@ -153,6 +158,18 @@ public void onFailure(Throwable t) {
     return promise;
   }
 
+  /**
+   * Transform the results of two {@link ListenableFuture}s a and b once both has completed using
+   * {@code function} and complete the returned ListenableFuture at that point. This method is
+   * synchronous in the sense that function.apply() executes synchronously and returns a value
+   * and not a {@link ListenableFuture}.
```

Comment:
I would rephrase all this as:

Transform the two input futures (a and b) into a single future, using the provided transform function. The transformation follows the same semantics as as Futures.transform and the two input futures are combined using Futures.allAsList.